sbids/tasks/lotkavolterra.py

```diff
@@ -0,0 +1,178 @@
+import jax
+from jax import lax
+import jax.numpy as jnp
+from jax.experimental.ode import odeint
+from tensorflow_probability.substrates import jax as tfp
+tfd = tfp.distributions
+tfb = tfp.bijectors
+
+
+class Lotka_Volterra:
+
+    def __init__(self,
+                num_seed:int=None):
```

Comment:
This is not good design for several reasons:
1. If a num_seed is necessary, then it shouldn't be left with None default. As a user, the first thing I tried was to instantiate the model like this "LV = Lotka_Volterra(); LV.get_truth()", which only at the second step triggered an error complaining that seed cannot be None.
2. In JAX code, each random function should explicitly be a function of an rng key. You do not want to have separate pieces of the code instantiate their own "jax.random.PRNGKey(self.num_seed)". This should happen only once, at the beginning of the JAX script, and all subsequent random function calls should use a key derived from that top key